src/main/java/tech/sourced/siva/SivaReader.java

```diff
@@ -0,0 +1,48 @@
+package tech.sourced.siva;
+
+import java.io.*;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+
+public class SivaReader {
+
+    private final RandomAccessFile sivaFile;
+    private final FileChannel channel;
+
+    public SivaReader(File sivaFile) throws FileNotFoundException {
```

Comment:
I think in this case is worth it, because If the file is not there is not a SivaReader problem, is a bug into the code that is creating the SivaReader using a file that does not exists. I think it should be a different error.